app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -1,6 +1,8 @@
 module ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::TowerApi
   extend ActiveSupport::Concern
 
+  class UnsupportedManagerOperation < StandardError; end
```

Comment:
I would expect all ManageIQ error classes to be a `MiqException` and to be defined here: https://github.com/ManageIQ/manageiq-gems-pending/blob/master/lib/gems/pending/util/miq-exception.rb